sympy/printing/latex.py

```diff
@@ -2449,7 +2449,7 @@ def _print_Parallel(self, expr):
         args = list(expr.args)
         parens = lambda x: self.parenthesize(x, precedence_traditional(expr),
                                              False)
-        return ' '.join(map(parens, args))
+        return '+'.join(map(parens, args))
```

Comment:
@namannimmo10, In my opinion, we should change this function. First of all, it is not printing equivalent to `expr`'s pretty form, and secondly, it adds unnecessary parentheses. It is just the Series' latex parsing function. If you can, suggest any good fix.

![image](https://user-images.githubusercontent.com/53227127/124648579-0e949f00-deb5-11eb-9315-bda6afca9868.png)

compared to
```
                                       [   3      ]   
                                       [5*s    s  ]
[5]                                    [----   -  ]
[-]                                    [ 1     1  ]
[1]                                    [          ]
[ ]    /[      3]      [      3]   \   [         2]
[5]    |[5  6*s ]      [5  6*s ]   |   [ 5    5*s ]
[-]    |[-  ----]      [-  ----]   |   [ -    ----]
[1]{t}*\[1   1  ]{t} + [1   1  ]{t}/ + [ 1     1  ]{t}
```
For Series too we should consider adding dot instead of space.

Reply:
**Update:**

Latex

![image](https://user-images.githubusercontent.com/53227127/125252799-63ea0980-e316-11eb-874f-02e357d34400.png)

Pretty-Print

```
                                       [   3      ]   
                                       [5*s    s  ]   
[5]                                    [----   -  ]   
[-]    /[      3]      [      3]   \   [ 1     1  ]   
[1]    |[5  6*s ]      [5  6*s ]   |   [          ]   
[ ]   *|[-  ----]    + [-  ----]   | + [         2]   
[5]    \[1   1  ]{t}   [1   1  ]{t}/   [ 5    5*s ]   
[-]                                    [ -    ----]   
[1]{t}                                 [ 1     1  ]{t}
```